GitUpKit/Views/Base.lproj/GICommitSplitterViewController.xib

```diff
@@ -46,7 +46,7 @@
                             <rect key="frame" x="127" y="8" width="855" height="16"/>
                             <autoresizingMask key="autoresizingMask" widthSizable="YES" flexibleMinY="YES"/>
                             <textFieldCell key="cell" controlSize="small" lineBreakMode="truncatingTail" allowsUndo="NO" sendsActionOnEndEditing="YES" alignment="left" title="&lt;TITLE&gt;" id="2Ta-mo-RyA">
-                                <font key="font" metaFont="controlContent"/>
+                                <font key="font" metaFont="cellTitle"/>
```

Comment:
feel like this shouldn't have been changed? Probably need to revert this line?

I just dragged and swapped the panels in the xib editor in xCode